app/enhance-app.js

```diff
@@ -49,19 +49,23 @@ const Error = {
   ) {
     return (
       <div>
-        <h1>{error.code}</h1>
-        <div>{error.message}</div>
-        <router-link to="/">Go Home!</router-link>
+        <h1>
+          {error.code}: {error.message}
+        </h1>
+        {__DEV__ &&
+          error.code === 404 &&
+          error.errorPath === '/' && (
+            <p>You must create pages/*.vue or export "router" in entry file!</p>
```

Comment:
is this message necessary? I mean it's obviously missing `router` or `pages` when there's 404 error

Reply:
Also note that it's only printed for `/`. In dev mode, this most probably means a misconfigured install.